toolchain-msvc.cmake

```diff
@@ -39,6 +39,9 @@ if(USE_CLANG_CL)
         set(CMAKE_C_COMPILER_TARGET "arm64-pc-windows-msvc")
         set(CMAKE_CXX_COMPILER_TARGET "arm64-pc-windows-msvc")
     else()
+        # -m32 is required for x64 clang-cl to operate in x86 Native Tools environment
```

Comment:
I don't quite understand this comment. In x86 native tools, clang-cl ahould be x86 and not require any options, it should only be needed in the x64_x86 cross tools environment.

Reply:
Well my assumption was that clang-cl is used from the VS installation, which should not need this. If this is to support an external installation, I'm fine with it, as long as it works for all environments and it looks like it does.